src/styles/partials/template/_sidebar.scss

```diff
@@ -10,15 +12,23 @@
   p {
     font-family: $default-heading-font;
     font-size: $smallest-heading-font-size;
+    margin: $default-margin 0;
   }
 
   h2 {
     border: none;
     letter-spacing: 10px;
     font-weight: $font-bold;
+    font-size: $h3-font-size;
```

Comment:
In TFS @sdurphy suggested this should be 18px.

Reply:
changed to 18